<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(185, 185, 185);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Titulo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#f50e0e;&quot;&gt;Polinomio Interpolador de Lagrange&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Grafica">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver gráfica </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Polinomio">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver polinomio expandido</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Lista">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver lista de datos</string>
    </property>
   </widget>
   <widget class="QLabel" name="Datos_X">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Datos en X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Datos_Y">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Datos en Y</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DigDatX">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DigDatY">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calcular">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="Escudo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/cct/Escudo UNAL.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Mos_Polinomio">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Polinomio_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Polinomio</string>
    </property>
   </widget>
   <widget class="QLabel" name="Evaluacion">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluación</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Mos_Evaluacion">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DigDatY_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="P_evaluar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Punto a evaluar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Next">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Siguiente Valor</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Escudo.qrc"/>
 </resources>
 <connections/>
</ui>
